.github/workflows/release.yaml

```diff
@@ -0,0 +1,39 @@
+name: Release
+on:
+  push:
+    tags:
+      - 'v*'
+permissions:
+  contents: read
+  pull-requests: read
+
+jobs:
+  tag_fix:
+    name: Prepare image tag
+    runs-on: ubuntu-22.04
+    outputs:
+      fixed_tag: ${{ steps.fix.outputs.tag }}
+    env:
+      TAGNAME: ${{ github.ref_name }}
+    steps:
+    - id: fix
+      # remove first character (v) and keep the next 12
```

Comment:
what's 12 for?

Reply:
Reason is silly. I thought that one had to set both start and end indices, but apparently it works fine with just `${TAGNAME:1}`